packages/nextjs/src/config/loaders/proxyLoader.ts

```diff
@@ -34,11 +36,25 @@ export default async function proxyLoader(this: LoaderThis<LoaderOptions>, userC
     // homepage), sub back in the root route
     .replace(/^$/, '/');
 
+  // For the `excludedServersideEntrypoints` option we need the calculate the relative path to the file in question without file extension.
+  const relativePagePath = path
+    .join('pages', path.relative(pagesDir, this.resourcePath))
+    // Pull off the file extension
+    .replace(new RegExp(`\\.(${pageExtensionRegex})`), '');
```

Comment:
m: do we need to worry about windows paths here?

Reply:
Yes and that should be taken care of by this implementation.